src/components/InputV2/Base.tsx

```diff
@@ -79,8 +78,7 @@ export const BaseInput = ({
   </Label>
 )
 
-export interface BaseInputProps
-  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'onChange'> {
```

Comment:
I wouldn't add all of the HTML input attributes.

Apart from onChange, we were already specifying other props that overlapped with the attributes (`disabled`, `placeholder`, `autoFocus`, etc.)

But mostly, the root element of this component is not an HTML input elements and there would be quite a few attributes that typescript would then allow to pass that we are not using / passing to our input element at all.

Since we're just adding the `autoComplete`, I think it'd be best to do just add that now. We can change it to accept all intput attributes but then we'd have to use rest props and object spread them into the `input`